src/device/usbd.c

```diff
@@ -463,7 +463,7 @@ static bool process_control_request(uint8_t rhport, tusb_control_request_t const
 {
   usbd_control_set_complete_callback(NULL);
 
-  TU_ASSERT(p_request->bmRequestType_bit.type < TUSB_REQ_TYPE_INVALID);
+  TU_VERIFY(p_request->bmRequestType_bit.type < TUSB_REQ_TYPE_INVALID);
```

Comment:
this is illegal request, host won't ever send this. It normally signals an issue with dcd getting incorrect or corrupted setup packet.